libcxx/CMakeLists.txt

```diff
@@ -784,6 +794,10 @@ config_define_if_not(LIBCXX_ENABLE_WIDE_CHARACTERS _LIBCPP_HAS_NO_WIDE_CHARACTER
 config_define_if_not(LIBCXX_ENABLE_TIME_ZONE_DATABASE _LIBCPP_HAS_NO_TIME_ZONE_DATABASE)
 config_define_if_not(LIBCXX_ENABLE_VENDOR_AVAILABILITY_ANNOTATIONS _LIBCPP_HAS_NO_VENDOR_AVAILABILITY_ANNOTATIONS)
 
+if (LIBCXX_LIBC STREQUAL "llvm-libc")
+  config_define(ON _LIBCPP_HAS_LLVM_LIBC)
+endif()
```

Comment:
I am not going to block this patch on fixing this, but we don't want to add a new `__config_site` definition every time we add support for a new library / platform. Instead, we should provide control over the implementation of localization used by the library at the CMake level, or a similar solution to prevent the proliferation of boolean flags.

Reply:
I removed these and will address it more correctly in a follow up PR.